tailwind.config.js

```diff
@@ -52,7 +52,10 @@ module.exports = {
         "gray-color": "#C4C4C4",
         "dark-gray": "#242423",
         "medium-gray": "#6B7280",
+        "gray-warmer": "#A4A4A4",
```

Comment:
Why do we have so many colors? e.g. gray-medium and medium-gray?  What is wrong with gray-300 and gray-900 ?

Reply:
removed new colors